### ISSUE TITLE
SQL highlighting between begin-select and end-select 

### ISSUE THREAD

**Issue by Mitch9378** [whanwells/language-sqr#3]
Add SQL syntax highlighting between begin and end select.

ie.

begin-select
**from** table a
**where** <some criteria>
**and** <some criteria>
end-select

The bold text would be properly highlighted. 

I have been trying to think about the best way to go about this. I think it would provide a lot of benefit. I am hopping to document the thought process here and anyone who is interest or has any ideas can add their input as well.

**Comment by whanwells** [comment]
This should be possible, but I'll have to look at some other grammar files to see how they accomplish something similar.

**Comment by Mitch9378** [comment]
Agreed I can't seem to find it now but I recall coming across a stack overflow on this specific topic. I link it here when I do find.
